spec/tapioca/cli/check_shims_spec.rb

```diff
@@ -344,10 +359,6 @@ class Bar; end
         end
 
         it "detects duplicates from Sorbet's payload" do
-          @project.write("sorbet/rbi/shims/core/object.rbi", <<~RBI)
-            class Object; end
-          RBI
```

Comment:
This used to be an error because the payload defines it as `class Object < BasicObject`